docker-compose.yml

```diff
@@ -190,26 +252,31 @@ services:
       DOCKER_VERNEMQ_ASTARTE_VMQ_PLUGIN__AMQP__HOST: "rabbitmq"
       DOCKER_VERNEMQ_USER_appengine: "appengine"
       CFSSL_URL: "http://cfssl:8080"
-    ports:
-      # 1883 is used internally and isn't exposed
-      # Used with HAProxy only
-      - "1885:1885"
-      # Used with SSL Listener only
-      - "8883:8883"
-      # You can comment this out if you don't need Let's Encrypt
-      - "80:80"
-      # HTTP Listener for health check
-      - "8888:8888"
+    # ports:
+    #   # HTTP Listener for health check
+    #   - "8888:8888"
+    #   # 1883 is used internally and isn't exposed
+    #   # Used with HAProxy only
+    #   - "1885:1885"
```

Comment:
I wouldn't rely on a docker compose comment for documentation. I would remove this from the file and rely on our doc to retrieve the same information

Reply:
There's a bunch of documentation comments here and there in many docker-compose-related files (e.g. `compose.env`). Shall we remove all of them in another PR?